system/jlib/jstats.cpp

```diff
@@ -1053,6 +1083,16 @@ StatisticMeasure queryMeasure(StatisticKind kind)
     return SMeasureNone;
 }
 
+bool doAggregateStat(StatisticKind kind)
```

Comment:
style:  do is a prefix for a function that does something.  Use a prefix like query/get/is/has for functions that return a boolean, especially if they are pure.